h/templates/activity/search.html.jinja2

```diff
@@ -211,163 +211,161 @@
 </section>
 {% endmacro %}
 
-{% macro sidebar(title, description) %}
+{% macro group_sidebar(group, group_edit_url, total, stats) %}
 <aside class="search-result-sidebar {%- if not more_info %} search-result-hide-on-small-screens{% endif %}">
```

Comment:
I'm struggling a little to follow what is now intentionally different in the top-level layout for the user and group sidebar. Can you clarify?